Orange/widgets/visualize/owmosaic.py

```diff
@@ -152,7 +152,6 @@ def state_count(self):
         Return the number of combinations, starting with a single attribute
         if Mosaic is colored by class distributions, and two if by Pearson
         """
-        self.compute_attr_order()
```

Comment:
Fixed. I also removed some dead code. It has not been used since changing color (class) was introduced.